app/src/test/java/org/mozilla/focus/browser/integration/FullScreenIntegrationTest.kt

```diff
@@ -95,61 +99,58 @@ internal class FullScreenIntegrationTest {
         val activity: Activity = mock()
         doReturn(activityWindow).`when`(activity).window
         doReturn(decorView).`when`(activityWindow).decorView
-        val integration = FullScreenIntegration(
-            activity, mock(), null, mock(), mock(), mock(), mock(), mock()
+
```

Comment:
Adding a newline here will break the `act-arrange-assert` structure of the test so it's probably best to avoid it.